iothub/service/src/ServiceClient.cs

```diff
@@ -99,111 +100,75 @@ public static ServiceClient CreateFromConnectionString(string connectionString,
-
-        /// <summary>
-        /// Interactively invokes a method on device
-        /// </summary>
-        /// <param name="deviceId">Device Id</param>
-        /// <param name="cloudToDeviceMethod">Device method parameters (passthrough to device)</param>
-        /// <param name="cancellationToken">Cancellation Token</param>
-        /// <returns>Method result</returns>
-        public abstract Task<CloudToDeviceMethodResult> InvokeDeviceMethodAsync(string deviceId, CloudToDeviceMethod cloudToDeviceMethod, CancellationToken cancellationToken);
+        /// <param name="cancellationToken">A cancellation token to cancel the operation.</param>
+        /// <returns>The service statistics that can be retrieved from IotHub, eg. the number of device connected to the hub.</returns>
+        public abstract Task<ServiceStatistics> GetServiceStatisticsAsync(CancellationToken cancellationToken = default);
 
         /// <summary>
-        /// Interactively invokes a method on device
+        /// Interactively invokes a method on a device.
         /// </summary>
-        /// <param name="deviceId">Device Id</param>
-        /// <param name="moduleId">Module Id</param>
-        /// <param name="cloudToDeviceMethod">Device method parameters (passthrough to device)</param>
-        /// <returns>Method result</returns>
-        public abstract Task<CloudToDeviceMethodResult> InvokeDeviceMethodAsync(string deviceId, string moduleId, CloudToDeviceMethod cloudToDeviceMethod);
+        /// <param name="deviceId">The device identifier for the target device.</param>
+        /// <param name="cloudToDeviceMethod">Device method parameters (passthrough to the device).</param>
+        /// <param name="cancellationToken">A cancellation token to cancel the operation.</param>
+        /// <returns>Method result.</returns>
+        public abstract Task<CloudToDeviceMethodResult> InvokeDeviceMethodAsync(string deviceId, CloudToDeviceMethod cloudToDeviceMethod, CancellationToken cancellationToken = default);
 
         /// <summary>
-        /// Interactively invokes a method on device
+        /// Interactively invokes a method on a module.
         /// </summary>
-        /// <param name="deviceId">Device Id</param>
-        /// <param name="moduleId">Module Id</param>
-        /// <param name="cloudToDeviceMethod">Device method parameters (passthrough to device)</param>
-        /// <param name="cancellationToken">Cancellation Token</param>
-        /// <returns>Method result</returns>
-        public abstract Task<CloudToDeviceMethodResult> InvokeDeviceMethodAsync(string deviceId, string moduleId, CloudToDeviceMethod cloudToDeviceMethod, CancellationToken cancellationToken);
+        /// <param name="deviceId">The device identifier for the target device.</param>
+        /// <param name="moduleId">The module identifier for the target module.</param>
+        /// <param name="cloudToDeviceMethod">Method parameters (passthrough to the module).</param>
```

Comment:
Wondering if we need to say - passthrough to the device. Is this better "Parameters to execute a direct method on the module"?